### PR TITLE
[CSUB-1061] Fix EVM balance was not added to Balance command total

### PR DESCRIPTION
**Pull request by pLabarta**
# Description of proposed changes

<describe what this PR is about and why we want it>

---

Practical tips for PR review & merge:

- [ ] All GitHub Actions report PASS
- [ ] Newly added code/functions have unit tests
  - [ ] Coverage tools report all newly added lines as covered
  - [ ] The positive scenario is exercised
  - [ ] Negative scenarios are exercised, e.g. assert on all possible errors
  - [ ] Assert on events triggered if applicable
  - [ ] Assert on changes made to storage if applicable
- [ ] Modified behavior/functions - try to make sure above test items are covered
- [ ] Integration tests are added if applicable/needed

### CODE REVIEW

File: cli/src/commands/balance.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ async function balanceAction(options: OptionValues) {
     const evmAddress = substrateAddressToEvmAddress(address);
     const evmBalance = new BN((await getEVMBalanceOf(evmAddress, getEvmUrl(options))).ctc.toString());
     balance.evm = evmBalance;
+    balance.total = balance.total.add(evmBalance);
```
Comment (atodorov) [inline cli/src/commands/balance.ts:30]: Ideally this should be `saturatingAdd()`, if available as part of the typescript libraries.  Otherwise LGTM.

**Review by atodorov — COMMENTED** [review]
Added a comment.

File: cli/src/commands/balance.ts (hunk 1 above)
Comment (atodorov) [reply cli/src/commands/balance.ts:30]: @pLabarta ping ^^^

Comment (pLabarta) [reply cli/src/commands/balance.ts:30]: Sadly, there is no Saturating Addition in the BN.js lib

**Review by atodorov — APPROVED** [review]

**Review by beqaabu — APPROVED** [review]

**Pull request merged**